scripts/provisioning-tests

```diff
@@ -40,8 +40,11 @@ eval "$(grep '^ENV CATTLE_CSI_PROXY_AGENT' package/Dockerfile | awk '{print "exp
 eval "$(grep '^ENV CATTLE_KDM_BRANCH' package/Dockerfile | awk '{print "export " $2 "=" $3}')"
 
 # Get the last release for $DIST, which is usually the latest version or an experimental version.
+# Previously this would use channels, but channels no longer reflect the latest version since
+# https://github.com/rancher/rancher/issues/36827 has added appDefaults. We do not use appDefaults
+# here for simplicity's sake, as it requires semver parsing & matching. The last release should
+# be good enough for our needs.
 export SOME_K8S_VERSION=$(curl -sS https://raw.githubusercontent.com/rancher/kontainer-driver-metadata/dev-v2.7/data/data.json | jq -r ".$DIST.releases[-1].version")
```

Comment:
I'm not sure if this is a solveable problem (and not something to worry about yet) but eventually we will branch for v2.8, and we will need to make sure we are constrained according to the following (example) constraints for the latest release:
```
    "maxChannelServerVersion": "v2.7.99",
    "minChannelServerVersion": "v2.7.2-alpha1",
```
This is not a problem yet, but definitely something to consider.

Reply:
```json
  "releases": [
   {
    "maxChannelServerVersion": "v2.6.99",
    "minChannelServerVersion": "v2.6.0-alpha1",
    "version": "v1.18.20+k3s1"
   },
   {
    "maxChannelServerVersion": "v2.6.99",
    "minChannelServerVersion": "v2.6.0-alpha1",
    "version": "v1.19.16+k3s1"
   },
   {
    "maxChannelServerVersion": "v2.6.99",
    "minChannelServerVersion": "v2.6.0-alpha1",
    "version": "v1.20.15+k3s1"
   },
```

I'm not sure I understand what's in this. These versions look out of date.

Reply:
If you scroll down to the bottom, you should see this:
```
    "maxChannelServerVersion": "v2.6.99",
    "minChannelServerVersion": "v2.6.7-alpha1",
...
    "version": "v1.24.8+rke2r1"
```
So I think it should be fine, when branching, but emergency patch releases are still something to consider.